### DISCUSSION TITLE
Run on bun

### DISCUSSION THREAD

**Discussion by kentcdodds** [epicweb-dev/epic-stack#685] [Ideas]
With Bun 1.1, windows is now supported and bcrypt as well. I think it's time for an [example](https://github.com/epicweb-dev/epic-stack/blob/main/docs/examples.md).

**Comment by koikar** [comment]
Hi @kentcdodds,
I'm excited to contribute by sharing my project, [tedi-stack](https://github.com/koikar/tedi-stack), which showcases a React Router 7 (RR7) setup with Bun and Hono server. My goal is to provide a starter template that can help facilitate the migration to Bun.

I’m currently using this setup in production, with authentication and many other features. So far, I haven't experienced any critical issues, but I would love to get feedback to improve this setup and ensure it follows best practices.

Initially I tried to create an example from the epic-stack, but ran into various issues and decided to take a more minimal approach as the first step.

As this is my very first open-source project, your insights would be invaluable as I work to refine the project and better serve the community.

Looking forward to your thoughts!

**Reply by kentcdodds** [reply]
I don't have a ton of time to evaluate the code, but the README looks really awesome. You've got a lot of valuable pieces in there. Really cool. Keep up the good work.

**Reply by koikar** [reply]
I already imagined, since migrating to Bun + switching to Hono is quite a significant undertaking for the epic-stack. Next I will add MCP with OAuth to see if I run into any issues in the Bun runtime. Btw, a thousand thanks for all your inspiring work, I'm following epic ai closely 🙌
